Job "Build resctl-demo" failing (time crate fails to build)

resctl-demo v2.2.5 (crates.io) uses time v0.3.34, which fails to build on rust >= 1.8.0 (https://github.com/rust-lang/rust/issues/127343). A simple workaround would be to pin the rust toolchain version to 1.79.0 until a new resctl-demo version is released and picks a more recent version of the time crate.

Failed job example: https://github.com/iocost-benchmark/iocost-benchmarks-ci/actions/runs/10552235255/job/29230770720

```
error[E0282]: type annotations needed for `Box<_>`
  --> /home/runner/.cargo/registry/src/index.crates.io-6f17d22bba15001f/time-0.3.34/src/format_description/parse/mod.rs:83:9
   |
83 |     let items = format_items
   |         ^^^^^
...
86 |     Ok(items.into())
   |              ---- type must be known at this point
   |
help: consider giving `items` an explicit type, where the placeholders `_` are specified
   |
83 |     let items: Box<_> = format_items
   |              ++++++++

   Compiling rand_chacha v0.1.1
   Compiling rand_pcg v0.1.2
   Compiling backtrace v0.3.69
   Compiling num-complex v0.2.4
   Compiling matrixmultiply v0.3.8
For more information about this error, try `rustc --explain E0282`.
error: could not compile `time` (lib) due to 1 previous error
warning: build failed, waiting for other jobs to finish...
```

--- a/.github/workflows/build-resctl-demo.yml
+++ b/.github/workflows/build-resctl-demo.yml
@@ -20,7 +20,7 @@ jobs:
         uses: actions-rs/toolchain@v1
         with:
           profile: minimal
-          toolchain: stable
+          toolchain: 1.79.0
           override: true
 
       - name: Checkout resctl-demo repository

